examples/dt/pidone/nncp/values.yaml

```diff
@@ -196,6 +203,36 @@ data:
         gateway: 10.0.0.1
         name: subnet1
     mtu: 1500
+  swift:
+    dnsDomain: swift.example.com
+    subnets:
+      - allocationRanges:
+          - end: 172.22.0.250
+            start: 172.22.0.100
+        cidr: 172.22.0.0/24
+        gateway: 172.22.0.1
+        name: subnet1
+        vlan: 25
+    mtu: 1500
+    prefix-length: 24
+    iface: swift
+    vlan: 25
+    base_iface: enp6s0
+    lb_addresses:
+      - 172.22.0.80-172.22.0.90
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "swift",
+        "type": "macvlan",
+        "master": "swift",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.22.0.0/24",
+          "range_start": "172.22.0.100",
+          "range_end": "172.22.0.250"
+        }
+      }
```

Comment:
afaicu this is needed otherwise not having swift in NetConfig would generate an error? I just crammed everything together instead of having a dedicated  networking/netconfig/kustomization.yaml since we don't expect to have multiple dts using this as a base.

Reply:
Since this is a brand-new block (`swift`) in the `values.yaml` here, you have complete control over what you place inside, since no pre-existing `kustomization.yaml` would have been referencing it.  It ultimately doesn't matter if you have extra unused data here, but it might be confusing to a reader who is trying to figure out where the data is referenced elsewhere (when some of it isn't at all).